arangod/Pregel/Conductor.h

```diff
@@ -46,14 +47,49 @@
 #include "Pregel/Conductor/StoringState.h"
 #include "Pregel/Status/ConductorStatus.h"
 #include "Pregel/Status/ExecutionStatus.h"
+#include "Pregel/WorkerInterface.h"
 #include "velocypack/Builder.h"
 
 #include <chrono>
+#include <cstdint>
 #include <memory>
 
 namespace arangodb {
 namespace pregel {
 
+struct ClusterConductorWorker : NewIWorker {
```

Comment:
Please, explain in a short comment what this class is for: it's an API of a worker whose instances run on conductors in a cluster.

Reply:
What do you think about a better name instead: ClusterWorkerOnConductor or ClusterWorkerOnCoordinator

Reply:
WorkerAPIOnWhatever or WorkerInterfeaceOnWhatever. "Whatever" is probably "Conductor", but that's not that important.